### PR TITLE
Choose default global provider for non-EIP-6963 dapps

### PR DESCRIPTION
**Pull request by everdimension**

### CODE REVIEW

File: src/ui/components/PauseInjection/actions.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    } else {
+      return now.add(1, 'day').hour(9).valueOf(); // 9AM next day
+    }
+  } else if (duration === TurnOffDuration.forever) {
+    return null;
+  }
+  throw new Error('Unexpected duration enum');
+}
+
+export function createPreference(
```
Comment (zerts) [inline src/ui/components/PauseInjection/actions.ts:36]: Let's name this method more specific. The same for `disablePreference`
May be `createPausePreference`

File: src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                  <span>Continue with Zerion</span>
+                </HStack>
+              </Button>
+              <Button
+                kind="regular"
+                onClick={async () => {
+                  await setGlobalPreferencesAsync(
+                    createPreference(globalPreferences, {
+                      origin,
+                      duration: TurnOffDuration.forever,
```
Comment (zerts) [inline src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx:73]: Do we want make this param const here?

File: src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                    })
+                  );
+                  onReject();
+                }}
+              >
+                Use Other (will reload dapp)
```
Comment (zerts) [inline src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx:79]: Did we finalise this text here?

File: src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+              You can always change this later inside Zerion Wallet
+            </UIText>
+          </VStack>
+        </div>
+        <div style={{ textAlign: 'center' }}>
+          <UIText
+            kind="small/accent"
+            color="var(--neutral-500)"
+            as={UnstyledAnchor}
+            href="https://zerion.io/blog/"
```
Comment (zerts) [inline src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx:96]: Do not forget to add valid link here

File: src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx (hunk 2 above)
Comment (everdimension) [reply src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx:73]: Yeah, I think giving a choice would clutter the UI significantly

File: src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx (hunk 4 above)
Comment (everdimension) [reply src/ui/pages/RequestAccounts/ChooseGlobalProvider/ChooseGlobalProvider.tsx:96]: We removed this from the UI ¯\\\_(ツ)_/¯

**Pull request merged**